src/extension.ts

```diff
@@ -83,7 +83,8 @@ export function activate(context: vscode.ExtensionContext): void {
         const siteType = util.isSiteDeploymentSlot(node.treeItem.site) ? 'Deployment Slot' : 'Web App';
         outputChannel.show();
         outputChannel.appendLine(`Starting ${siteType} "${node.treeItem.site.name}"...`);
-        await node.treeItem.siteWrapper.start(nodeUtils.getWebSiteClient(node));
+        await node.treeItem.start(nodeUtils.getWebSiteClient(node));
+        node.treeDataProvider.refresh(node, false);
```

Comment:
I thought this didn't work in the released version of VS Code - don't you still need to refresh the parent?

In any case, I would prefer if you used `node.refresh()` and then we only have to change it in one place (aka [here](https://github.com/Microsoft/vscode-azuretools/blob/master/ui/src/treeDataProvider/AzureNode.ts#L62)) once VS Code releases the tree fix